testsuite/integration-tests/pom.xml

```diff
@@ -45,6 +45,12 @@
         <container.qualifier.manual.ssl.wrong>jbossas-manual-ssl-wrong</container.qualifier.manual.ssl.wrong>
         <container.qualifier.manual.ssl.wildcard>jbossas-manual-ssl-wildcard</container.qualifier.manual.ssl.wildcard>
         <container.qualifier.manual.ssl.sni>jbossas-manual-ssl-sni</container.qualifier.manual.ssl.sni>
+        
+        <!-- For grpc-bridge test -->
+        <version.jaxrs.example.grpc>0.0.2-SNAPSHOT</version.jaxrs.example.grpc>
+        <version.org.wildfly.core>18.1.0.Final</version.org.wildfly.core>
+        <version.weld.environment>5.0.0.CR2</version.weld.environment>
+        <version.wildfly.weld>26.1.0.Final</version.wildfly.weld>
```

Comment:
This should be removed.

Reply:
The last three are gone. <version.jaxrs.example.grpc> is for the archetype generated project that is used specifically in the arquillian test GrpcToJaxrsTest, so I think it makes sense to leave it here.